util/host/main.lua

```diff
@@ -1,22 +1,3 @@
-print("\n\n") --give us some room
-
-function usage()
-  print([[
-    
-    Moai Host Utility. Run from project root folder (outside lua source)
-    
-    Usage: moaiutil host <subcommand> <args>
-    
-    Subcommands:
-      host list - Lists available hosts
-      host init - Creates a template host config file used by subsequent commands
-      host build <hostname> - Creates (if it doesn't exist) in hosts folder, and (re)builds the host named <hostname>. 
-      host run <hostname> - Creates (if it doesn't exists) in hosts folder, (re)builds and runs the host named <hostname>. 
-      host create <hostname> - Creates the host in the hosts folder (removing old host) based on latest config settings.
-    ]])
```

Comment:
I think that it is still good to have command usage info, so I don't have to wade through tonnes of irrelevant usage info to find out the command to build a host